rmgpy/thermo/thermodata.pyx

```diff
@@ -402,3 +402,21 @@ cdef class ThermoData(HeatCapacityModel):
         :class:`NASAPolynomial` objects.
         """
         return self.to_wilhoit().to_nasa(Tmin, Tmax, Tint, fixedTint, weighting, continuity)
+
+    cpdef bint is_all_zeros(self):
+        """
+        Check whether a ThermoData object has all zero values, e.g.: 
+        
+            ThermoData(
+                Tdata=([300, 400, 500, 600, 800, 1000, 1500], "K"),
+                Cpdata=([0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], "J/(mol*K)"),
+                H298=(0.0, "kJ/mol"),
+                S298=(0.0, "J/(mol*K)"),
+            )
+    
+        Returns:
+            bool: Whether all values are zeroes or not.
+        """
+        if self.H298.value_si == 0.0 and self.S298.value_si == 0.0 and all([cp == 0.0 for cp in self.Cpdata.value_si]):
+            return True
+        return False
```

Comment:
Great addition and thanks for asking me to review :) Did you ever have problems with some, but not all, of the values being 0? Should that feature be added as an additional check?

Reply:
I thought @mjohnson541 commented in yesterday's meeting that these groups were probably intended to be populated (original authors though they are important)

Reply:
Sorry, I see now that you meant "some but not all". Then, no, I didn't have problems like that, if that's the case then it indeed makes things more complicated.

Reply:
Okay yea just wanted to check if that was a case we should also consider, but I didn't know if it was a common enough problem. My understanding from yesterday was that the "all zeros case" was affecting others as well so thanks for the fix!